translate/storage/php.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, source=""):
         """Construct a blank phpunit."""
         self.escape_type = None
         super(phpunit, self).__init__(source)
-        self.name = ""
+        self.name = "$text"
```

Comment:
I would use something that clearly states that this is a placeholder, like `$TTK_PLACEHOLDER` or similar.

Reply:
Good point, will change this.